lib/api/core/plugins/pluginsManager.js

```diff
@@ -165,7 +166,6 @@ class PluginsManager {
       debug(
         '[%s] starting plugin in "%s" mode',
         plugin.manifest.name,
-        plugin.config.privileged ? 'privileged' : 'standard');
 
       return Bluebird
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Parse error at line 170 column 7:

160:           pipeWarnTime,
161:           pipeTimeout,
162:           initTimeout
163:         } = this.config.common;
164: 
165: 
166:       debug(
167:         '[%s] starting plugin in "%s" mode',
168:         plugin.manifest.name,
169: 
170:       return Bluebird
           ^
171:         .resolve(plugin.object.init(
172:           plugin.config,
173:           plugin.config.privileged
174:             ? new PrivilegedPluginContext(this.kuzzle, plugin.manifest.name)
175:             : new PluginContext(this.kuzzle, plugin.manifest.name)))
176:         .timeout(initTimeout)
177:         .then(initStatus => {
178:           if (initStatus === false) {
179:             errorsManager.throw(
180:               'plugins', [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AParsingError)